CMakeLists.txt

```diff
@@ -697,6 +697,11 @@ endif()
 add_subdirectory(Externals/imgui)
 add_subdirectory(Externals/implot)
 add_subdirectory(Externals/glslang)
+add_subdirectory(Externals/Lua)
+find_package (Python3 COMPONENTS Interpreter Development)
+if(NOT Python3_FOUND)
+  message(STATUS "Using static cpython from Externals")
+  add_subdirectory(Externals/cpython)
```

Comment:
Typo, there needs to be an `endif()` after this line. Other than that, everything seems to build correctly via cmake in WSL Ubuntu for me.